src/test/java/uk/gov/companieshouse/web/pps/util/PPSTestUtility.java

```diff
@@ -44,9 +46,14 @@ public static LateFilingPenalty validLateFilingPenalty(String ID) {
     public static PayableLateFilingPenalty validPayableLateFilingPenalty(String companyNumber, String ID) {
         PayableLateFilingPenalty payableLateFilingPenalty = new PayableLateFilingPenalty();
         payableLateFilingPenalty.setCompanyNumber(companyNumber);
+
+        Payment payment = new Payment();
+        payment.setPaidAt(DATE_TIME);
+        payment.setAmount(VALID_AMOUNT.toString());
         String resumeURI = "/late-filing-penalty/company/" + companyNumber + "/penalty/" + ID + "/view-penalties";
 
         payableLateFilingPenalty.setLinks(new HashMap<String, String>(){{put("resume_journey_uri", resumeURI);}});
```

Comment:
'new HashMap<String, String>() ' cane be 'new HashMap<>()'. Are you using the SonarLint plugin? This should pick this up.

Reply:
Update as suggested